vm/src/stdlib/os.rs

```diff
@@ -1027,6 +1027,47 @@ fn os_chroot(path: PyStringRef, vm: &VirtualMachine) -> PyResult<()> {
     nix::unistd::chroot(path.as_str()).map_err(|err| convert_nix_error(vm, err))
 }
 
+// As of now, redox does not seems to support chown command (cf. https://gitlab.redox-os.org/redox-os/coreutils , last checked on 05/07/2020)
+#[cfg(all(unix, not(target_os = "redox")))]
+fn os_chown(
+    path: PyPathLike,
+    uid: PyIntRef,
+    gid: PyIntRef,
+    dir_fd: DirFd,
+    follow_symlinks: FollowSymlinks,
+    vm: &VirtualMachine,
+) -> PyResult<()> {
+    let uid = isize::try_from_object(&vm, uid.as_object().clone())?;
+    let gid = isize::try_from_object(&vm, gid.as_object().clone())?;
+
+    let uid = if uid >= 0 {
+        Some(nix::unistd::Uid::from_raw(uid as u32))
+    } else if uid == -1 {
+        None
+    } else {
+        return Err(vm.new_os_error(String::from("Specified uid is not valid.")));
+    };
+
+    let gid = if gid >= 0 {
+        Some(nix::unistd::Gid::from_raw(gid as u32))
+    } else if gid == -1 {
+        None
+    } else {
+        return Err(vm.new_os_error(String::from("Specified gid is not valid.")));
+    };
+
+    let flag = match follow_symlinks.follow_symlinks {
+        true => nix::unistd::FchownatFlags::FollowSymlink,
+        false => nix::unistd::FchownatFlags::NoFollowSymlink,
+    };
```

Comment:
This should fix the error in CI.
```suggestion
    let flag = if follow_symlinks.follow_symlinks {
        nix::unistd::FchownatFlags::FollowSymlink
    } else {
        nix::unistd::FchownatFlags::NoFollowSymlink
    };
```